src/main/java/nextstep/subway/domain/line/Line.java

```diff
@@ -0,0 +1,165 @@
+package nextstep.subway.domain.line;
+
+import nextstep.subway.domain.section.Section;
+import nextstep.subway.domain.section.Sections;
+import nextstep.subway.domain.station.Station;
+import nextstep.subway.domain.station.Stations;
+import nextstep.subway.error.exception.BusinessException;
+import nextstep.subway.error.exception.ErrorCode;
+
+import javax.persistence.*;
+import java.util.ArrayList;
+import java.util.Collections;
+
+@Entity
+public class Line {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    private String name;
+    private String color;
+
+    @Embedded
+    private Sections sections = new Sections();
+
+    public Line() {
+    }
+
+    public Line(String name, String color) {
+        this.name = name;
+        this.color = color;
+    }
+
+    public Line(Long id, String name, String color) {
+        this.id = id;
+        this.name = name;
+        this.color = color;
+    }
+
+    public Long getId() {
+        return id;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public String getColor() {
+        return color;
+    }
+
+    public Sections getSections() {
+        return sections;
+    }
+
+    public Stations getStations() {
+        if (this.sections.getList().isEmpty()) {
```

Comment:
[디미터 법칙](https://johngrib.github.io/wiki/law-of-demeter/) 을 참고해보시면 어떨까요?

```suggestion
        if (this.sections.isEmpty()) {
```

Reply:
아하 아예 외부에서 접근하지 못하도록 캡슐화를 해야했던거군요..!!
제가 정확하게 디미터 법칙을 이해하지 못했던 것 같습니다!
이렇게 수정해볼게요!